core/src/main/kotlin/ch/ergon/dope/resolvable/expression/unaliased/type/numeric/NumberFunctionExpression.kt

```diff
@@ -0,0 +1,21 @@
+package ch.ergon.dope.resolvable.expression.unaliased.type.numeric
+
+import ch.ergon.dope.DopeQuery
+import ch.ergon.dope.resolvable.expression.TypeExpression
+import ch.ergon.dope.resolvable.operator.FunctionOperator
+import ch.ergon.dope.validtype.NumberType
+
+sealed class NumberFunctionExpression(
+    private val symbol: String,
+    private val value: TypeExpression<NumberType>,
+    private val extraValue: TypeExpression<NumberType>? = null,
```

Comment:
can you merge _value_ and _extraValue_ into varargs of type _TypeExpression<NumberType>?_ (like in the ArrayFunctionExpression) and filter out the null-values before calling _toDopeQuery_?

Reply:
please also remove the extra-variable from the _ArrayFunctionExpression_ and change the type of _args_ to nullable (and rename the param to _arguments_)

Reply:
done